ee/models/event_definition.py

```diff
@@ -6,17 +6,20 @@
 
 
 class EnterpriseEventDefinition(EventDefinition):
-    owner = models.ForeignKey("posthog.User", null=True, on_delete=models.SET_NULL, related_name="event_definitions",)
+    owner = models.ForeignKey("posthog.User", null=True, on_delete=models.SET_NULL, related_name="event_definitions")
     description: models.TextField = models.TextField(blank=True, null=True, default="")
     updated_at: models.DateTimeField = models.DateTimeField(auto_now=True)
     updated_by = models.ForeignKey("posthog.User", null=True, on_delete=models.SET_NULL, blank=True)
     verified: models.BooleanField = models.BooleanField(default=False, blank=True)
     verified_at: models.DateTimeField = models.DateTimeField(null=True, blank=True)
     verified_by = models.ForeignKey(
-        "posthog.User", null=True, on_delete=models.SET_NULL, blank=True, related_name="verifying_user"
+        "posthog.User", null=True, on_delete=models.SET_NULL, blank=True, related_name="verifying_user",
     )
 
     # Deprecated in favour of app-wide tagging model. See EnterpriseTaggedItem
     deprecated_tags: ArrayField = deprecate_field(
         ArrayField(models.CharField(max_length=32), null=True, blank=True, default=list), return_instead=[],
     )
+    tags: ArrayField = deprecate_field(
+        ArrayField(models.CharField(max_length=32), null=True, blank=True, default=list), return_instead=[],
```

Comment:
I'm pretty sure this is a blocking migration that we want to avoid (cc @EDsCODE )